aspnetcore/tutorials/choose-web-ui.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Choose an ASP.NET Core UI
+author: wadepickett
+description: Understand when to use the various ASP.NET Core web UI technologies Microsoft provides and supports.
+ms.author: wpickett
+ms.date: 06/30/2021
+no-loc: [Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: tutorials/choose-web-ui
+---
+# Choose an ASP.NET Core web UI
+
+ASP.NET Core is a complete end to end UI framework.  You choose which functionalities to combine that fit your web UI app needs.
+
+Start by determining if your web UI app will primarily benefit from pre-rendering UI on the server, or rendering UI at the client browser. Then review the server rendered or client rendered ASP.NET Core project types to find which best fit your apps needs. 
+
+## Choose an ASP.NET Core web UI technology
+
+Use the following flowchart to select an ASP.NET Core web UI technology:
+
+![Decision tree for ASP.NET Core web UI technology](choose-web-ui/_static/choose_ui.png)
```

Comment:
I don't think this decision tree adds much value in its current form. The branch conditions aren't very descriptive and it makes it look  like working with ASP.NET Core is complicated. As much as possible we want users to feel like there is one  happy path solution for any web UI scenario that they want to implement.

Reply:
@danroth27, Thanks for the feedback!   Should I beef up branch descriptions and try to make it more helpful, or remove it?  I felt it helped provide a picture of what you need to choose from and when, and  to get a handle quickly on the choices. 
 That said I personally find a visual map very helpful for sorting out the larger picture in my brain, perhaps more than the average developer.

Reply:
I think we're better off removing it. I don't see an easy path for clarifying the graphic.

Reply:
Removed graphic, thanks.